examples/learn_bivariate_normal.py

```diff
@@ -22,7 +22,7 @@
 def learn_bivariate_normal() -> None:
     # Lazily instantiated flow plus base and target distributions
     bijectors = bij.AffineAutoregressive(
-        params=params.DenseAutoregressive(hidden_dims=(32,))
+        hypernet=params.DenseAutoregressive(hidden_dims=(32,))
```

Comment:
I recommend we change `hypernet` to `params_fn` since a hypernet is a very specific thing: a neural net that outputs the parameters of another neural net. Here, `params_fn` does not have to be a NN and the output doesn't have to be used in a NN either